tools/sgen/input/FluentUIStyle.yml

```diff
@@ -92,11 +92,12 @@ Colors:
   Presence:
     - available: "NamedColor(presenceAvailable)"
     - away: "NamedColor(presenceAway)"
+    - awayOutOfOffice: "NamedColor(presenceOof)"
     - blocked: "NamedColor(presenceBlocked)"
     - busy: "NamedColor(presenceBusy)"
     - doNotDisturb: "NamedColor(presenceDnd)"
     - offline: "NamedColor(presenceOffline)"
-    - outOfOffice: "NamedColor(presenceOof)"
+    - offlineOutOfOffice: "NamedColor(presenceOof)"
     - unknown: "NamedColor(presenceUnknown)"
```

Comment:
Only away and offline need a separate "OOF" entry? and why don't these have to line up exactly with the Avatar presence enum?

Reply:
As far as colors go, Away and Offline are the only ones that change.
The icons have a different mapping though.

What Teams passed as a requirement is that each item of the presence menu should be rendered in a certain way for both "Regular" or "OOF" modes.

Away and Offline should render as the "Out Of Office" icon.
The other ones are just a regular icon as opposed to a filled one (and it's not for all of the status).